pandas/tests/indexing/test_interval.py

```diff
@@ -172,6 +269,12 @@ def test_non_unique(self):
         expected = s.iloc[0:1]
         tm.assert_series_equal(expected, result)
 
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
+    def test_non_unique_updated_behavior(self):
+
+        # Actually I think we should remove this test? Not sure what exactly it's meant to gauge...
```

Comment:
I'm also not entire sure. I don't think this test needs to change though -- the existing test looks fine.